pro/src/screens/Offers/NoResults/NoResults.tsx

```diff
@@ -22,7 +27,13 @@ const NoResults = ({ resetFilters }: INoResultsProps): JSX.Element => (
     <p className={styles['search-no-results-text']}>
       Vous pouvez modifer votre recherche ou
       <br />
-      <Link className="reset-filters-link" onClick={resetFilters} to="/offres">
+      <Link
+        className="reset-filters-link"
+        onClick={resetFilters}
+        to={`/offres${
```

Comment:
Il me semble que tu à une url pour ca.
```suggestion
        to={`/offres/collective'}
```